pkg/api/validate/dynamic/quota.go

```diff
@@ -9,68 +9,86 @@ import (
 	"net/http"
 
 	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/api/v20210131preview"
```

Comment:
Preview API includes the set of VMs in `pkg/api` as well as the new vm types in `pkg/api/v20210131preview` that is necessary for compute isolation.  Add required resources will use the same types with the addition of the new types added to the preview api.